spec/support/helpers.rb

```diff
@@ -48,7 +48,7 @@ def stub_api_v2_request(url_path, stub, method = :get, status = 200, token: nil,
           "Accept-Encoding" => "gzip,deflate",
           "Authorization"   => "Bearer #{token}",
           "Content-Type"    => "application/vnd.api+json",
-          "User-Agent"      => "Faraday v0.15.4",
+          "User-Agent"      => "Faraday v0.17.0",
```

Comment:
Any way to get this out of Faraday?

Reply:
worry about it next time?

Reply:
https://github.com/lostisland/faraday/blob/master/lib/faraday.rb#L22

> ```
> module Faraday
>   VERSION = '0.16.0'
> ```

SRSLY??

Reply:
rails dev :man_facepalming: :rofl:

Reply:
why do we need to set it anyway?

Reply:
@defong because it's comparing the full json payload

@misaka found that `Faraday::VERSION` matches so I can do another PR for using that